src/go/rpk/pkg/cli/cmd/common/common_test.go

```diff
@@ -351,106 +414,261 @@ func TestKafkaAuthConfig(t *testing.T) {
 }
 
 func TestBuildTLSConfig(t *testing.T) {
+	truststoreContents := `-----BEGIN CERTIFICATE-----
+MIIDDjCCAfagAwIBAgIUZHm6D1aJYtmVEV1X23m+oK9EJgcwDQYJKoZIhvcNAQEL
+BQAwLTETMBEGA1UECgwKVmVjdG9yaXplZDEWMBQGA1UEAwwNVmVjdG9yaXplZCBD
+QTAeFw0yMTA0MDYwMjMzMjNaFw0yMjA0MDYwMjMzMjNaMC0xEzARBgNVBAoMClZl
+Y3Rvcml6ZWQxFjAUBgNVBAMMDVZlY3Rvcml6ZWQgQ0EwggEiMA0GCSqGSIb3DQEB
+AQUAA4IBDwAwggEKAoIBAQDZN7ytDxWGvxprVvydv0hwuD3hGcZFbJaeDXrIInGd
+RGK8zr/zoq3oAgw1ZW0OabID3OKs9tezKGM5wPUvzGfU94qhi1ot040Utw+Jf2tQ
+GA3T1X+VHRTUlqDVVnIvhcAiy21bMUMVuFl4QtJnWx9ZljkCFo8IIh/3Sq88ORDl
+gzfM3cK1kk+60uKzXNvgK8ShrZ0GYsbmxncxhbdr2O7mSdVcO6x0tbmNDLc5Hx+w
+34z2hRHReavw3KDFfaIdHZE2tSQ4xh25F9aZQiPsTaGzPtGozLl7Ck1p9Ew4d+MO
+EXd51gGeIdP6EUD8exOMfRq2B6/p18HyjMsuKQKhjx2NAgMBAAGjJjAkMA4GA1Ud
+DwEB/wQEAwIC5DASBgNVHRMBAf8ECDAGAQH/AgEBMA0GCSqGSIb3DQEBCwUAA4IB
+AQCse+GLdlBi77fOgDgX8dby2PVZiSoW8cE1uRJzaXvw9mrz83RW2SxnW3och6Mm
+cixyv+taomZfYNM2wbOzEpkI0QEcV9CF/9Sx5RQVKsoAQkjkmzpHUeRp5ha/VAYq
+KWVCj9Ej+1y8dE0+AvltyymRbcMKUSk3vXK6HmzCGn2XAVz1WBQHQHXe0vwQGAXu
+Tq7VoNh+LNEud7ro5Hwi5aiDA1B7HZum6u2MvU5KwGY3txDve1Jn0bWOa8J3HjIN
+wHAv/4PAPweXflPAVHkUR4VOslBdZo/tAcSbG/Zr3cneBt7VYnPyO+IAe7S54u9g
+eKlljKHOgHw7gXOzsvWTVQbm
+-----END CERTIFICATE-----`
+
+	certContents := `Certificate:
```

Comment:
This line all the way to `-----BEGIN CERTIFICATE-----` are not needed

Reply:
Maybe use golden files? in the `test_data` folder?

Reply:
Aren't golden files to compare output?

Reply:
Ok my bad. `golden files` are for comparing output. The `testsdata` folder is meant for test fixtures/ancillary data used by tests.

REF:
* https://dave.cheney.net/2016/05/10/test-fixtures-in-go
* https://golang.org/pkg/cmd/go/internal/test/

> The go tool will ignore a directory named "testdata", making it available
to hold ancillary data needed by the tests.